index.d.ts

```diff
@@ -0,0 +1,128 @@
+declare module 'fluture' {
+  interface ResolveFunction<T> {
+    (value: T): void
+  }
+  interface RejectFunction<S> {
+    (error: S): void
+  }
+
+  interface FunctionReturning<T> {
+    (...t: any[]): T
+  }
+  abstract class Future<T, S> {
+    done<T, S> (fn: (err: S, value: T) => void): Future<T, S>
```

Comment:
I thought we are only going to add typings for `fork` ? 
https://github.com/fluture-js/Fluture/pull/129#discussion_r127650946